db/bigtable_eth1.go

```diff
@@ -2559,11 +2559,11 @@ func (bigtable *Bigtable) GetAddressErc1155TableData(address string, search stri
 
 	tableData := make([][]interface{}, len(transactions))
 	for i, t := range transactions {
-		from := utils.FormatHash(t.From)
+		from := utils.FormatAddressWithLimits(t.From, "", false, "", 11, 0, false)
 		if fmt.Sprintf("%x", t.From) != address {
 			from = utils.FormatAddressAsLink(t.From, "", false, false)
 		}
-		to := utils.FormatHash(t.To)
+		to := utils.FormatAddressWithLimits(t.To, "", false, "", 11, 0, false)
```

Comment:
The same issue as in the [comment](https://github.com/gobitfly/eth2-beaconchain-explorer/pull/1951/files/2bd8c741533cffed6a7eca31a73020ace94c5d9f..05aaeceedccc39f58282ec26095b389364c9ae2d#r1140068198) above.

Reply:
fixed

Reply:
> * erred" line

It seems the data (incl. formatting) is cached therefor the changes don't take effect emmidatly.

You could circumenvent the cache by adding a +"1" after the cacheKey in line 31 of the eth1data.go . Start the app reload. And it should work. Now when you remove the +"1" start and reload it should have the new version in your cache.